src/dc_federated/algorithms/fed_avg/fed_avg_server.py

```diff
@@ -58,15 +57,7 @@ class FedAvgServer(object):
         This is mandatory if ssl_enabled is True, ignored otherwise.
     """
 
-    def __init__(self,
-                 global_model_trainer,
-                 key_list_file,
-                 update_lim=10,
-                 server_host_ip=None,
-                 server_port=8080,
-                 ssl_enabled=False,
-                 ssl_keyfile=None,
-                 ssl_certfile=None):
+    def __init__(self, server_port, global_model_trainer, key_list_file, update_lim=10, server_host_ip=None, ssl_enabled=False, ssl_keyfile=None, ssl_certfile=None):
```

Comment:
@techjacker Why did you move this parameter forward?

Reply:
My thinking was to make it a required rather than optional parameter as it will be fed in via the helm chart and to avoid debugging issues arising from a default port being set.